docs/reference/docs/delete.asciidoc

```diff
@@ -105,7 +105,8 @@ thrown instead.
 [[delete-index-creation]]
 === Automatic index creation
 
-The delete operation automatically creates an index if it has not been
+If external version is used,
```

Comment:
I'm fine with something generic here (if one of the external versioning variants is used) and linking to the versioning page.